packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropsWithChildren } from 'react';
```

Comment:
This is the `ModalAutocomplete` which I've just taken a couple things and put them in types/utils to reuse them in `AutocompleteList`